source/Calamari/Commands/ExecuteManifestCommand.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Autofac.Features.Metadata;
+using Calamari.Commands.Support;
+using Calamari.Common.Commands;
+using Calamari.Common.Plumbing.Variables;
+using Calamari.LaunchTools;
+using Calamari.Serialization;
+using Newtonsoft.Json;
+
+namespace Calamari.Commands
+{
+    [Command("execute-manifest")]
+    public class ExecuteManifestCommand : Command
+    {
+        readonly IVariables variables;
+        readonly IEnumerable<Meta<ILaunchTool, LaunchToolMeta>> executionTools;
+
+        public ExecuteManifestCommand(
+            IVariables variables,
+            IEnumerable<Meta<ILaunchTool, LaunchToolMeta>> executionTools)
+        {
+            this.variables = variables;
+            this.executionTools = executionTools;
+        }
+
+        public override int Execute(string[] commandLineArguments)
+        {
+            Options.Parse(commandLineArguments);
+
+            var contents = variables.Get("Octopus.Steps.Manifest");
```

Comment:
The manifest file is actually sent via a variable so that is encrypted since it can contain sensitive values